### PR TITLE
20.03 -> 20.09 bump

### LINKED ISSUE #416: Download Page: Link to GNOME ISO
It's a thing now https://github.com/NixOS/nixpkgs/pull/85711.
We should do this when we release the next NixOS: 20.09.

### PR DESCRIPTION
**Pull request by garbas**
## TODO

- [x] release announcement (highlight of the release)
- [x] release banner (with expiration time, eg. 1 month)
- [x] Add gnome flavor iso to the download page
- [x] check that download page links work (all ISO, EC2 images) and correct version is displayed in the header
- [x] check that correct manual versions are displayed for nixos and nixpkgs

closes #416
closes https://github.com/NixOS/nixos-homepage/issues/628

This change is [Reviewable](https://reviewable.io/reviews/nixos/nixos-homepage/619)

### CODE REVIEW

**Review by jonringer — APPROVED** [review]
LGTM

Once the release announcement is done

File: news.xml [hunk 1]
```diff
@@ -7,7 +7,99 @@
...
+      </p>
+
+      <ul>
+        <li>7349 packages were added,</li>
+        <li>8181 packages were removed,</li>
+        <li>14442 packages were updated.</li>
+      </ul>
+
+      <p>
+        Likewise our NixOS module system got bigger and reacher:
```
Comment (cole-h) [inline news.xml:68]:
```suggestion
        Likewise, our NixOS module system increased in size:
```

Or something.

Comment (jonringer) [reply news.xml:68]: yea, i saw that as well, I'm going to work on this after finishing the mark broken stuff

File: news.xml [hunk 2]
```diff
@@ -1,6 +1,107 @@
...
+      </ul>
+
+      <p>
+        <strong>TODO: @jon - change number of contributors and commits.</strong>
+        The 20.09 release was possible because of work of <strong>400</strong>
+        contributors in <strong>20000</strong> commits. We would especially
+        like to thank our top 10 contributors:
+        <a href="https://github.com/">NameA</a>,
+        <a href="https://github.com/">NameB</a>,
+        ...
```
Comment (samueldr) [inline news.xml:47]: Hi, friendly reminder of a TODO :)

File: news.xml [hunk 3]
```diff
@@ -1,6 +1,107 @@
...
+        on time, we prefer to make a release with a better quality.
+      </p>
+
+      <p>
+        <strong>TODO: @jon - add contributors who helped</strong>
+        Individuals who contributed to stabilizing this release:
+        <a href="https://github.com/">NameA</a>,
+        <a href="https://github.com/">NameB</a>,
+        ...
+      </p>
```
Comment (samueldr) [inline news.xml:92]: Hi :wave: friendly reminder again here :)

File: news.xml [hunk 4]
```diff
@@ -1,6 +1,107 @@
...
+
+      <h3>Reflections and Closing</h3>
+
+      <p>
+        # TODO: @jon >>
+        - Talk about being a release manager for 20.09 Jon and working with @worldofpeace etc.
+        - Thank co-release-manager @worldofpeace
+        - Mention Risicle / thanks
+        - Thank everyone in general/closing
+      </p>
```
Comment (samueldr) [inline news.xml:102]: :wave: here too!

**Review by samueldr — CHANGES_REQUESTED** [review]
Just in case, since I was checking the PR, I highlighted your TODO items.

Everything else looks alright at a glance.

**Comment by jonringer** [comment]
thanks @samueldr, working on it now

File: news.xml (hunk 3 above)
Comment (jonringer) [reply news.xml]: added

File: news.xml (hunk 4 above)
Comment (jonringer) [reply news.xml]: added

File: news.xml (hunk 2 above)
Comment (jonringer) [reply news.xml]: added

**Review by samueldr — APPROVED** [review]

**Comment by samueldr** [comment]
(Not that you needed my approval)

File: news.xml [hunk 5]
```diff
@@ -1,6 +1,119 @@
...
+        <a href="https://github.com/worldofpeace">WORLDofPEACE</a>,
+        <a href="https://github.com/flokli">Florian Klink</a>,
+        <a href="https://github.com/romildo">José Romildo Malaquias</a>.
+      </p>
+
+      <p>
+        NixOS is already known as
+        <a href="https://repology.org/repositories/statistics/newest">the most up to date distribution</a>
+        and is
+        <a href="https://repology.org/repositories/statistics/total">at the top three by total number of packages</a>.
```
Comment (cole-h) [inline news.xml:60]:
```suggestion
        <a href="https://repology.org/repositories/statistics/total">in the top three by total number of packages</a>.
```

File: news.xml [hunk 6]
```diff
@@ -1,6 +1,119 @@
...
+      </p>
+
+      <p>
+        This didn't stop us. In last six months:
```
Comment (cole-h) [inline news.xml:64]:
```suggestion
        This didn't stop us. In the last six months:
```

File: news.xml [hunk 7]
```diff
@@ -1,6 +1,119 @@
...
+        <a href="https://github.com/mkg20001">Maciej Krüger</a>.
+      </p>
+
+      <h3>Reflections and Closing</h3>
+
+      <p>
+        I think that the 20.09 release highlighted a few weak points with our current release schedule.
+        Discussions have already began on how to improve the process from the beginning, to help minimize
+        risk and better set ourselves up for success. I do want to thank <a href="https://gihtub/worldofpeace">WORLDofPEACE</a>
+        for helping me with release management items. I also wanted to thank <a href="https://github.com/ttuegel">Thomas Tuegel</a> for
```
Comment (cole-h) [inline news.xml:112]:
```suggestion
        for helping me with release management items. I also want to thank <a href="https://github.com/ttuegel">Thomas Tuegel</a> for
```

File: news.xml [hunk 8]
```diff
@@ -1,6 +1,119 @@
...
+        <a href="https://github.com/jonringer">Jonathan Ringer</a>,
+        <a href="https://github.com/mkg20001">Maciej Krüger</a>.
+      </p>
+
+      <h3>Reflections and Closing</h3>
+
+      <p>
+        I think that the 20.09 release highlighted a few weak points with our current release schedule.
+        Discussions have already began on how to improve the process from the beginning, to help minimize
+        risk and better set ourselves up for success. I do want to thank <a href="https://gihtub/worldofpeace">WORLDofPEACE</a>
```
Comment (cole-h) [inline news.xml:111]:
```suggestion
        risk and better set ourselves up for success. I do want to thank <a href="https://github.com/worldofpeace">WORLDofPEACE</a>
```

File: news.xml [hunk 9]
```diff
@@ -1,6 +1,119 @@
...
+        <li>1119 options were added (61 new modules),</li>
+        <li>476 options removed,</li>
+        <li>118 options were updated.</li>
+      </ul>
+
+      <h3>Stabilization Contributors</h3>
+
+      <p>
+        Stabilization of the NixOS happens a month before planned release. The
+        goal is to get as little as possible continuous integrations (Hydra)
```
Comment (cole-h) [inline news.xml:87]:
```suggestion
        goal is to get as little as possible continuous integration (Hydra)
```

File: news.xml [hunk 10]
```diff
@@ -1,6 +1,119 @@
...
+        jobs failing before the release is cut. While we would like to release
+        on time, we prefer to make a release with a better quality.
```
Comment (cole-h) [inline news.xml:89]:
```suggestion
        on time, a high quality release is more important.
```

Maybe?

File: news.xml [hunk 11]
```diff
@@ -1,6 +1,119 @@
...
+                href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09-notable-changes">Other
+                Notable Changes</a></li>
+          </ul>
+        </li>
+        <li><a href="https://nixos.org/manual/nixos/stable/index.html#sec-upgrading">Upgrade Instructions</a></li>
+        <li><a href="https://nixos.org/download.html#download-nixos">Image downloads</a></li>
+      </ul>
+
+      <p>
+        The 20.09 release was possible because of work of <strong>1313</strong>
```
Comment (cole-h) [inline news.xml:41]:
```suggestion
        The 20.09 release was possible due to the efforts of <strong>1313</strong>
```

**Review by cole-h — CHANGES_REQUESTED** [review]
A few minor nits for ya.

**Comment by worldofpeace** [comment]
@cole-h You are my number 1 angel :rofl: I'm wondering how many of this could be put back into the release notes.

File: news.xml [hunk 12]
```diff
@@ -1,6 +1,119 @@
...
+        and is
+        <a href="https://repology.org/repositories/statistics/total">in the top three by total number of packages</a>.
+      </p>
+
+      <p>
+        This didn't stop us. In the last six months:
+      </p>
+
+      <ul>
+        <li>7349 packages were added,</li>
```
Comment (davidak) [inline news.xml:68]: i think the `,` don't add any useful information and don't belong in such list. it's not a sentence

File: news.xml [hunk 13]
```diff
@@ -1,6 +1,119 @@
...
+        <li>8181 packages were removed,</li>
+        <li>14442 packages were updated.</li>
+      </ul>
+
+      <p>
+        Likewise our NixOS module system got bigger and better:
+      </p>
+
+      <ul>
+        <li>1119 options were added (61 new modules),</li>
```
Comment (davidak) [inline news.xml:78]: remove `,`

**Review by davidak — CHANGES_REQUESTED** [review]
very good overall :sparkles:

File: news.xml [hunk 14]
```diff
@@ -1,6 +1,119 @@
...
+      </p>
+
+      <ul>
+        <li>7349 packages were added,</li>
+        <li>8181 packages were removed,</li>
+        <li>14442 packages were updated.</li>
+      </ul>
+
+      <p>
+        Likewise our NixOS module system got bigger and better:
```
Comment (cole-h) [inline news.xml:74]:
```suggestion
        Likewise, our NixOS module system got bigger and better:
```

File: news.xml (hunk 12 above)
Comment (cole-h) [reply news.xml:68]: And remove the final period in the list.

File: news.xml (hunk 13 above)
Comment (cole-h) [reply news.xml:78]: Same as above.

**Review by cole-h — CHANGES_REQUESTED** [review]

**Comment by jonringer** [comment]
thanks @davidak @cole-h

**Review by davidak — APPROVED** [review]

**Review by cole-h — APPROVED** [review]
It's so close, I can *taste it*.

**Comment by jonringer** [comment]
I think that's it folks :)

**Comment by davidak** [comment]
I do the same for the manual release notes now.

**Comment by jonringer** [comment]
> I do the same for the manual release notes now.

those are already merged, as they are a part of nixpkgs

**Pull request merged**